interfaces/builtin/audio_playback.go

```diff
@@ -0,0 +1,177 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/interfaces/seccomp"
+	"github.com/snapcore/snapd/interfaces/udev"
+	"github.com/snapcore/snapd/release"
+	"github.com/snapcore/snapd/snap"
+)
+
+// The audio-playback interface is the companion interface to the audio-record
+// interface. The design of this interface is based on the idea that the slot
+// implementation (eg pulseaudio) is expected to query snapd on if the
+// audio-record slot is connected or not and the audio service will mediate
+// recording. If other audio recording servers require different security
+// policy for record and playback (eg, a different socket path), then those
+// accesses will be added to this interface.
+
+const audioPlaybackSummary = `allows operating as or interacting with playback audio services`
+
+const audioPlaybackBaseDeclarationSlots = `
+  audio-playback:
+    allow-installation:
+      slot-snap-type:
+        - app
+        - core
+    deny-connection:
+      on-classic: false
+`
+
+const audioPlaybackConnectedPlugAppArmor = `
```

Comment:
I actually expect them to differ. In fact, I cleaned up the accesses in audio-playback where they overlapped with alsa already (pulseaudio predated the alsa interface). Furthermore, I don't expect pulseaudio to git (m)any more updates because we want to deprecate its use entirely, but I do expect that we will update audio-*, eg for pipewire, etc.

Reply:
@zyga - put another way, I actually was going to do exactly what you suggested, but then after digging in I felt this would be cleaner, today. I then thought since we are planning on removing the pulseaudio interface at some future (20.N?) date, then having these audio-* interfaces not dependent on/intertwined with it would make that removal easier.